src/AutoRoller.sol

```diff
@@ -835,4 +845,28 @@ contract RollerUtils {
         Space _space = periphery.spaceFactory().pools(address(adapter), nextMaturity);
         return (_space, _space.getPoolId(), _space.pti(), adapter.scale());
     }
-}
+
+    function getNewTargetedRate(uint256 /* prevTargetedRate */, address adapter, uint256 prevMaturity, Space space) public returns (uint256) {
+        (, uint48 issuance, , , , , uint256 iscale, uint256 mscale, ) = DividerLike(DIVIDER).series(adapter, prevMaturity);
+
+        require(mscale != 0);
+
+        if (mscale <= iscale) return 0;
+
+        // Calculate the rate implied by the growth in scale over the previous Series term.
+        uint256 rate = (_powWad(
+            (mscale - iscale).divWadDown(iscale) + ONE, ONE.divWadDown((prevMaturity - issuance) * ONE)
```

Comment:
Consider rebranding `issuance` to `prevIssuance` for specificity and consistency with `prevMaturity`.

Reply:
added!